src/ui/settings.ts

```diff
@@ -26,6 +28,7 @@ export interface FullCalendarSettings {
         mobile: string;
     };
     timeFormat24h: boolean;
+    locale: string;
```

Comment:
```suggestion
    locale?: string;
```

Existing users will not have this locale setting set, so it will need to be added as optional to the settings type. After making this change, there will likely be typescript errors that will need to be fixed so that the code handles potentially undefined locale settings in older settings.